src/components/state/Headers/header.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import communityLanding from '../../../resources/img/community-landing.svg'
+import './style.css'
+import mainImage from '../../../resources/img/work-imgs/productivity.svg'
+
+class Header extends Component {
+    state = {  }
+    render() { 
+        return ( 
+            <div>
+                <div className = 'headers'>
+                    <h4 className="workwithux">WORK WITH UX SOCIETY</h4>
```

Comment:
Use the `section-label` class here.

Reply:
For the main div that encompasses the whole component or for the current "headers" div?

Reply:
On line 12, the h4 tag.